### DISCUSSION TITLE
Tool for labeling images with rotation

### DISCUSSION THREAD

**Discussion by Petopp** [open-mmlab/mmrotate#421] [General]
Hello all,
wanted to ask how best or with what you could do the labeling of the images. I have already seen that there is a version of labelimg which can specify the rotation to me. However, one would have to convert this data then again from XML to Dota. Is yes the recommendation.

Can someone a Toll empfhelen for this? Or has someone a converter from XML to DOTA already programmed in Python?

Thanks for the support already once

Peter

**Comment by javvi51** [comment]
I am using https://github.com/heshameraqi/labelImg_OBB, it works quite well and it gives you the label in a txt file with class,x,y,w,h,angle. I have coded my own function to convert that into the DOTA type.

**Reply by imadgohar** [reply]
labelimg2 can also be used https://github.com/chinakook/labelImg2. It is also very handy and you can get PASCAL VOC format annotations and then convert to DOTA.
